sass/_component_link.scss

```diff
@@ -1,6 +1,9 @@
 a {
     color: $link-color;
     text-decoration: 2px solid underline;
```

Comment:
We should drop this line now

Reply:
I wonder if that would have unintended side effects in other browsers

Reply:
I have checked on mdn web docs. All browsers support long hand versions. Only Firefox for Android doesn't support `text-decoration-thickness` neither in short nor in long hand way. So I think removing it will be safe.